### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Mkarthik2002**
testing 1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,6 @@ Why should you un-manage your work? You should use this snap-in if you want to r
 
 If your team’s best practice is to link your GitHub activity to an existing issue-ID during branch creation, commits and PRs, then this snap-in will honor that and enable a GitHub driven issue state, without auto creating issues. [Here](https://devrev.ai/docs/integrations/github#github-events) are the different ways you can link your DevRev issue to GitHub branches, commits and PRs.
 
-However, if you choose to work in GitHub and never created or linked an issue, you shouldn’t have to sweat it. This snap-in will automatically create issues that track all associated commits, branches and pull requests. The auto-created issue is enriched with PR description and related GitHub events. The status of these issues is driven by GitHub activity, and the issues will auto-transition stages, from In Development to In Review, and Closed. Your work is always accounted for and more importantly your team members have visibility and know what you’ve been up to, resulting in fewer interruptions.
 
 ## How it works
```
Comment (Mkarthik2002) [inline README.md:12]: add some changes

Comment (Mkarthik1234) [inline README.md:12]: can you change this

**Comment by Mkarthik1234** [comment]
how is this working

File: README.md [hunk 2]
```diff
@@ -8,7 +8,6 @@ Why should you un-manage your work? You should use this snap-in if you want to r
 
 If your team’s best practice is to link your GitHub activity to an existing issue-ID during branch creation, commits and PRs, then this snap-in will honor that and enable a GitHub driven issue state, without auto creating issues. [Here](https://devrev.ai/docs/integrations/github#github-events) are the different ways you can link your DevRev issue to GitHub branches, commits and PRs.
```
Comment (Mkarthik1234) [inline README.md:9]: remove this

Comment (Mkarthik1234) [inline README.md:9]: testing 1

**Comment by Mkarthik1234** [comment]
testing 3

**Pull request closed without merging**